### ISSUE TITLE
About relu_fn

### ISSUE THREAD

**Issue by weiyichang** [lukemelas/EfficientNet-PyTorch#27]
Hi,
Thanks for sharing your implementation.
I have a question about relu_fn in utils.py, you use x*sigmoid as activation function. 
Is it better than ReLU or  Swish ?
Just want to know that did you try different activation functions ?
Thanks!!! 

Update:
Sorry, Swish activation function in MobilenetV3 is different, and the original Swish is as your implementation.

**Comment by lukemelas** [comment]
Thanks for updating your comment. For others who stumble upon this issue, `relu_fn` is actually a swish activation function. It's defined here: https://github.com/lukemelas/EfficientNet-PyTorch/blob/master/efficientnet_pytorch/utils.py#L38 

I call it `relu_fn` to be consistent with the original Tensorflow implementation
